test/index.js

```diff
@@ -134,54 +132,52 @@ describe('index', function () {
       expect(args[2]).to.have.property('errStack');
     });
 
-    it('should log metadata', function () {
+    it('should log metadata', function() {
       log.debug('hi', { name: 'Dan' });
 
       var args = spy.args[0];
 
       expect(args[2]).to.have.property('name', 'Dan');
     });
-
   });
 
-  describe('middleware', function () {
-
-    it('should have middleware on the instance', function () {
+  describe('middleware', function() {
+    it('should have middleware on the instance', function() {
       var log = new index.Log({ transports: [], sentry: {} });
 
       expect(log).to.have.property('middleware');
       expect(log.middleware).to.be.a('function');
     });
 
-    it('should produce a middleware function', function () {
+    it('should produce a middleware function', function() {
       var log = new index.Log({ transports: [], sentry: {} });
       var middleware = log.middleware();
 
       expect(middleware).to.be.a('function');
       expect(middleware).to.have.length(3);
     });
-
   });
 
-  describe('_getSentryMeta', function () {
-
-    it('should return just tags if meta is not defined', function () {
+  describe('_getSentryMeta', function() {
+    it('should return just tags if meta is not defined', function() {
       var result = index._getSentryMeta();
 
-      expect(result).to.be.eql({
+      expect(result).to.deep.equal({
         extra: {},
         tags: {
-          env: 'test'
-        }
+          env: 'test',
+        },
+        fingerprint: undefined,
```

Comment:
Updated this test to pass now. Not sure what allowed it to pass previously without the `fingerprint` and `level` keys.